change/@ni-nimble-components-225009b9-77d6-4cbd-a94e-d9ba2751b047.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I prefer questions and mockups to be presented in person if you'd like input.  I want to avoid having to dig through threads as much as possible.

Reply:
I can present in person, if that's better for visual designers. However, it would be useful to have these reviewed by visual designers to ensure the information contains what they need from IxD. The intent is to have things documented to support asynchronous workflows and capture discussions about the design so we can go back and re-examine them as needed.